commit cdccd87f868a55cb782b9702e7eed311be18c466 (refs/changes/69/2369/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2017-08-22T21:21:58+00:00 (2 years, 2 months ago)
    
    ADMINUI-2314: Upgrade Node.js version from 0.10 to 4.x
    Reviewed by: Pedro P. Candel <pedro@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index 8d8da58..d8ff174 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -206,8 +206,11 @@ cat <<EOF
     "image_name": "adminui",
     "image_description": "SDC AdminUI",
     "image_version": "1.0.0",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
-    "pkgsrc": [],
+    "// image_uuid": "sdc-minimal-multiarch-lts@15.4.1",
+    "image_uuid": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
+    "pkgsrc": [
+      "patch-2.7.5"
+    ],
     "repos": [
         {"url": "git@github.com:joyent/sdc-adminui.git", "dir": "adminui"}
     ],
